react/index.js

```diff
@@ -147,7 +149,7 @@ class ProductSummary extends Component {
     return img
   }
 
-  get renderImageLoader() {
```

Comment:
Why remove the getters?

Reply:
because the getters doesn't make sense for a *function*